en/sql/query/delete.rst

```diff
@@ -102,3 +108,82 @@ The below queries delete rows after joining multiple tables. They show the same
     INNER JOIN c_tbl c ON b.id=c.id;
 
 For more details on join syntax, see :ref:`join-query`.
+
+You can also delete remote table data using the extended table name. The query below is for deleting remote table data.
+
+.. code-block:: sql
+
+    --at remote srv1
+    CREATE TABLE a_tbl(
+        id INT NOT NULL,
+        phone VARCHAR(10));
+    INSERT INTO a_tbl VALUES(1,'111-1111'), (2,'222-2222'), (3, '333-3333'), (4, NULL), (5, NULL);
+
+    --at local
+    --delete one record only from remote a_tbl
+    DELETE FROM a_tbl@srv1 WHERE phone IS NULL LIMIT 1;
+    SELECT * FROM a_tbl;
+               id  phone
+    ===================================
+                1  '111-1111'
+                2  '222-2222'
+                3  '333-3333'
+                5  NULL
+    --delete all records from remote a_tbl
+    DELETE FROM a_tbl@srv1;
+
+The tables below were created to explain remote DELETE JOIN.
+
+.. code-block:: sql
+
+    --at remote srv1
+    CREATE TABLE a_tbl(
+        id INT NOT NULL,
+        phone VARCHAR(10));
+
+    CREATE TABLE b_tbl(
+        id INT NOT NULL,
+        phone VARCHAR(10));
+
+    CREATE TABLE c_tbl(
+        id INT NOT NULL,
+        phone VARCHAR(10));
+
+    --at local
+    INSERT INTO a_tbl@srv1 VALUES(1,'111-1111'), (2,'222-2222'), (3, '333-3333'), (4, NULL), (5, NULL);
+    INSERT INTO b_tbl@srv1 VALUES(1,'111-1111'), (2,'222-2222'), (3, '333-3333'), (4, NULL);
+    INSERT INTO c_tbl@srv1 VALUES(1,'111-1111'), (2,'222-2222'), (10, '333-3333'), (11, NULL), (12, NULL);
+
+The following queries perform deletion after joining tables including remote tables (2 local tables, 1 remote table), and all show the same result.
+
+.. code-block:: sql
+
+    -- Below four queries show the same result.
+    --  <DELETE multiple tables FROM ...>
+
+    DELETE a, b FROM a_tbl@srv1 a, b_tbl@srv1 b, c_tbl@srv1 c
+    WHERE a.id=b.id AND b.id=c.id;
+
+    DELETE a, b FROM a_tbl@srv1 a INNER JOIN b_tbl@srv1 b ON a.id=b.id
+    INNER JOIN c_tbl@srv1 c ON b.id=c.id;
+
+    -- <DELETE FROM multiple tables USING ...>
+
+    DELETE FROM a, b USING a_tbl@srv1 a, b_tbl@srv1 b, c_tbl@srv1 c
+    WHERE a.id=b.id AND b.id=c.id;
+
+    DELETE FROM a, b USING a_tbl@srv1 a INNER JOIN b_tbl@srv1 b ON a.id=b.id
+    INNER JOIN c_tbl@srv1 c ON b.id=c.id;
+
+Caution
+==========
+
+As shown below, DELETE JOIN queries that include local tables and remote tables and delete remote tables are not allowed.
```

Comment:
```suggestion
As shown below, DELETE JOIN queries that include local tables and remote tables and delete remote tables are not allowed. This is because it does not send local data to the remote.
```